maintainers/maintainer-list.nix

```diff
@@ -11943,6 +11943,12 @@
     githubId = 9799623;
     name = "Rick van Schijndel";
   };
+  minersebas = {
```

Comment:
I think you should reword the commit to use the attribute name: `maintainers: add minersebas`.